codecov.yml

```diff
@@ -1,27 +0,0 @@
-coverage:
```

Comment:
we should keep this.

Reply:
So this was one of my questions--why is there _also_ a codecov.yml file in the .github directory?

Reply:
oh now I see the question in the description. I don't see a codecov.yml file in the github dir (?).

Reply:
Oh wait. You're totally right, will add it back. This is what I get for doing these things in the middle of the night!

Reply:
Oh, but I _can_ move it to `.github/`: https://github.com/cosmos/cosmos-sdk/pull/7828#discussion_r518337441 🎉